src/components/Actions/Actions.vue

```diff
@@ -302,7 +302,7 @@ export default {
 		},
 		focusAction() {
 			// TODO: have a global disabled state for non input elements
-			const focusElement = this.$refs.menu.querySelectorAll('.focusable:not(:disabled)')[this.focusIndex]
+			const focusElement = this.$refs.menu.querySelectorAll(`.focusable:not([class$='disabled'])`)[this.focusIndex]
```

Comment:
I would rather use this in case the browser add the disabled class before the focusable.
```suggestion
			const focusElement = this.$refs.menu.querySelectorAll(`.focusable:not([class*='disabled'])`)[this.focusIndex]
```

Reply:
Also, there are other refs.menu.querySelectorAll bellow :)